src/language-markdown/embed.js

```diff
@@ -13,7 +13,9 @@ function embed(path, print, textToDoc, options) {
   const node = path.getValue();
 
   if (node.type === "code") {
-    const parser = getParserName(node.lang);
+    // only look for the first string so as to support [markdown-preview-enhanced](https://shd101wyy.github.io/markdown-preview-enhanced/#/code-chunk)
+    const lang = node.lang.split(/\s/)[0];
```

Comment:
How about adding [`limit`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split) to `split`?

```js
const lang = node.lang.split(/\s/, 1)[0];
```

We only use `[0]` so there is no point in extracting all words from the string

Reply:
Sure 👍